### ISSUE TITLE
ToggleButton documentation does not mention BooleanProperty: allow_no_selection

### ISSUE THREAD

**Issue by DavidA2014** [kivy/kivy#3842]
ToggleButton documentation does not mention BooleanProperty: allow_no_selection 

http://kivy.org/docs/api-kivy.uix.togglebutton.html#

https://github.com/kivy/kivy/blob/master/kivy/uix/behaviors/togglebutton.py#L30

**Comment by DavidA2014** [comment]
Sorry, I now see that allow_no_selection is defined in the base class docs. But wouldn't it be better if inherited properties showed up in inheriting classes?

**Comment by tshirtman** [comment]
Yes, it's a matter of configuring sphinx for it, i believe, and it would indeed make reading easier, it's not like we are printing the pdf anyway.

**Comment by Zen-CODE** [comment]
Yup. The shinx autodoc extension has the option ':inherited-members:' for autoclass and automodule directives.

Although I'm not a fan of doing this as it makes the docs much more verbose, I can't deny that showing inherited methods is clearer, more complete and avoids many possible issues. I've seen that with our trainees. Will submit a PR for that :-)

**Comment by Zen-CODE** [comment]
PR merged

https://github.com/kivy/kivy/commit/61b9555352e27d13abe45088334f3ec8de8ac015

**Comment by dessant** [comment]
related discussion https://github.com/kivy/kivy/pull/3870
